packages/cli/test/unit/OptionsTest.ts

```diff
@@ -53,16 +53,6 @@ describe('Options', function () {
     deepEqual(config.sourcePaths, ['src/', 'a.js'])
   })
 
-  it('treats sources as globs', function () {
```

Comment:
I moved this test to `CLITest.ts`. That's a more end-to-end test, and is thus more robust in the face of changing internals.